src/components/item/settings/ThumbnailSetting.tsx

```diff
@@ -149,6 +137,18 @@ const ThumbnailSetting = ({ item }: Props): JSX.Element | null => {
           maxWidth={THUMBNAIL_SETTING_MAX_WIDTH}
           maxHeight={THUMBNAIL_SETTING_MAX_HEIGHT}
         />
+        <input
+          type="file"
+          accept="image/*"
+          onInput={onSelectFile}
+          // onChange is successfully triggered in test
```

Comment:
What about this comment? not sure what information it is. If it was a previous comment I guess you can remove it.